staging/vhost-device-gpu/src/main.rs

```diff
@@ -34,15 +36,20 @@ struct GpuArgs {
     /// vhost-user Unix domain socket.
     #[clap(short, long, value_name = "SOCKET")]
     socket_path: PathBuf,
+    /// Select the GPU mode: either 'gfxstream', 'virglrenderer', or '2d'
+    #[clap(short, long)]
+    renderer: String,
```

Comment:
You can do this easier:

You can create an enum lets say `RenderMode` with `#[derive(clap::ValueEnum)]` and put the desired variants in the enum. Then you can have `renderer: RenderMode` here directly and clap will automatically check if the argument the user provided is in the enum, and automatically translate the user string to the correct enum value. It will also print nice error messeges if the user provides wrong value of the argument.

Reply:
This also means you don't have to add `serde` dependency.